src/createAgentManager.ts

```diff
@@ -109,36 +151,52 @@ export async function createAgentManager(agent: string | Agent, options: AgentMa
     const baseURL = options.baseURL || didApiUrl;
     const wsURL = options.wsURL || didSocketApiUrl;
     const abortController: AbortController = new AbortController();
+    const mxKey = options.mixpanelKey || mixpanelKey;
+
     const agentsApi = createAgentsApi(options.auth, baseURL);
     const ratingsAPI = createRatingsApi(options.auth, baseURL);
     const knowledgeApi = createKnowledgeApi(options.auth, baseURL);
 
     const agentInstance = typeof agent === 'string' ? await agentsApi.getById(agent) : agent;
     options.callbacks?.onAgentReady?.(agentInstance);
 
+    const analytics = AnalyticsProvider.getInstance({
+        mixPanelKey: mxKey,
+        agent: agentInstance,
+    });
+
     const socketManager = await SocketManager(options.auth, wsURL, options.callbacks.onChatEvents);
-    let { chat, streamingManager } = await initializeStreamAndChat(agentInstance, options, agentsApi);
+    let { chat, streamingManager } = await initializeStreamAndChat(agentInstance, options, agentsApi, analytics);
+
+    analytics.track('agent-sdk-created', getInitAnaliticsInfo(agentInstance));
 
     return {
         agent: agentInstance,
         chatId: chat.id,
         async reconnectToChat() {
+            analytics.track('agent-resume-chat');
```

Comment:
add propd to event
and lets call it
agent-chat, event: resume

Reply:
please address this comment

Reply:
@o-bieliaiev

Reply:
done